src/components/dashboard/AllFiles.tsx

```diff
@@ -0,0 +1,162 @@
+import UploadIcon from '@mui/icons-material/Upload';
+import { Box, Chip, Link, Paper, Stack, Typography } from '@mui/material';
+import { format } from 'date-fns';
+import { compact } from 'lodash-es';
+import { useMemo, useState } from 'react';
+
+import FileDialog from './files/FileModal';
+import useFileActions from './files/useFileActions';
+
+import ButtonLink from '@/components/elements/ButtonLink';
+import { ColumnDef } from '@/components/elements/GenericTable';
+import useSafeParams from '@/hooks/useSafeParams';
+import GenericTableWithData from '@/modules/dataFetching/components/GenericTableWithData';
+import { useHasClientPermissions } from '@/modules/permissions/useHasPermissionsHooks';
+import { DashboardRoutes } from '@/routes/routes';
+import {
+  FileFieldsFragment,
+  GetClientFilesDocument,
+  GetClientFilesQuery,
+  GetClientFilesQueryVariables,
+  PickListType,
+  useGetPickListQuery,
+} from '@/types/gqlTypes';
+import generateSafePath from '@/utils/generateSafePath';
+
+const AllFiles = () => {
+  const { clientId } = useSafeParams() as { clientId: string };
+  const [viewingFile, setViewingFile] = useState<
+    FileFieldsFragment | undefined
+  >();
+
+  const { getActionsForFile, deleteFileDialog } = useFileActions({
+    onDeleteFile: () => setViewingFile(undefined),
+  });
+
+  const [canEdit] = useHasClientPermissions(clientId, [
+    'canManageAnyClientFiles',
+    'canManageOwnClientFiles',
+  ]);
+  const { data: pickListData } = useGetPickListQuery({
+    variables: { pickListType: PickListType.AvailableFileTypes },
+  });
+
+  const columns: ColumnDef<FileFieldsFragment>[] = useMemo(() => {
+    return [
+      {
+        header: 'Name',
```

Comment:
Wrapped titles get centered for some reason, can we have them left-aligned to match
<img width="1178" alt="Screen Shot 2023-03-24 at 4 46 17 PM" src="https://user-images.githubusercontent.com/5333982/227636260-62c017c3-defe-4523-b1e1-96d1e610f80d.png">